clients/admin-ui/src/features/config-wizard/AddSystemForm.tsx

```diff
@@ -84,25 +82,6 @@ const AddSystemForm = () => {
                 <QuestionIcon boxSize={5} color="gray.400" />
               </Tooltip>
             </Stack>
-            <Stack direction="row" display="flex" alignItems="center" mb={5}>
-              <IconButton
-                aria-label="Okta"
-                boxSize={iconButtonSize}
-                minW={iconButtonSize}
-                boxShadow="base"
-                variant="ghost"
-                icon={<OktaLogoIcon boxSize="full" />}
-                onClick={() => dispatch(changeStep())}
-              />
-              <Text>System Scan (Okta)</Text>
-              <Tooltip
-                fontSize="md"
-                label="System scanning allows you to connect to your sign-on platform and automatically identify systems that should be on your data map."
-                placement="right"
-              >
-                <QuestionIcon boxSize={5} color="gray.400" />
-              </Tooltip>
-            </Stack>
```

Comment:
If there's a ticket for adding okta, linking this diff to it could save some time when we get to it.